src/types/docs/function-properties-trigger.md

```diff
@@ -4,17 +4,20 @@
 
 one (and only one) of
 
-*   [Cron](function-properties-trigger-oneof-cron.md "check type definition")
+*   [Untitled  type in Function](function-properties-trigger-oneof-0.md "check type definition")
 
-*   [Frequecy](function-properties-trigger-oneof-frequecy.md "check type definition")
+*   [Untitled  type in Function](function-properties-trigger-oneof-1.md "check type definition")
```

Comment:
this looks autogenerated, but these "Untitled type in Function" items look a little weird, are they missing a field or something?

Reply:
Correct, they are missing a title field in the main schema. The problem is that adding them to oneOf (or required) attributes causes the serverless schema validation to fail. Haven't found a way around that yet. Fortunately, the files that are referenced in the Readme are fine.